net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Enterprise OID Format</fielddescr>
+			<fieldname>translate_enterprise_oid_format</fieldname>
+			<description>
+				<![CDATA[
+					Configures how the symbolic enterprise OID will be displayed for $E.<br/>
+					Set to 1, 2, 3 or 4.  See translate_value_oids options 1,2,3 and 4.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+				<option><name>3</name><value>3</value></option>
+				<option><name>4</name><value>4</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Trap OID Format</fielddescr>
+			<fieldname>translate_trap_oid_format</fieldname>
+			<description>
+				<![CDATA[
+					Configures how the symbolic trap OID will be displayed for $O.<br/>
+					Set to 1, 2, 3 or 4.  See translate_value_oids options 1,2,3 and 4.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+				<option><name>3</name><value>3</value></option>
+				<option><name>4</name><value>4</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Variable Name OID Format</fielddescr>
+			<fieldname>translate_varname_oid_format</fieldname>
+			<description>
+				<![CDATA[
+					Configures how the symbolic trap OID will be displayed for $v, $-n, $+n, $-* and $+*.<br/>
+					Set to 1, 2, 3 or 4.  See translate_value_oids options 1,2,3 and 4.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+				<option><name>3</name><value>3</value></option>
+				<option><name>4</name><value>4</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Integers</fielddescr>
+			<fieldname>translate_integers</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to disable converting INTEGER values to enumeration tags as defined in the MIB files.<br/>
+					Set to 1 to enable converting INTEGER values to enumeration tags as defined in the MIB files.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Wildcard Expansion Separator</fielddescr>
+			<fieldname>wildcard_expansion_separator</fieldname>
+			<description>
+				<![CDATA[
+					Set what is used to separate variables when wildcards are expanded on the FORMAT / EXEC line.<br/>
+					Defaults to a space.  Value MUST be within quotes. Can contain 1 or more characters.<br/>
```

Comment:
Why force the user to put in the quotes rather than doing so automatically in the code that makes the configuration? It seems like an unnecessary restriction. If it must stay this way, it must at least state if single or double quotes are necessary.